tests/Kernel/EntityRevisionWithTypeFieldTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation\Kernel;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+
+/**
+ * Tests the EntityRevisionWithType field type, widget and formatter.
```

Comment:
We are not really testing the widget though, we would need to use the form for that.